zio-http/src/test/scala/zhttp/http/RequestSpec.scala

```diff
@@ -0,0 +1,19 @@
+package zhttp.http
+
+import zio.test.Assertion._
+import zio.test._
+
+object RequestSpec extends DefaultRunnableSpec {
+  def spec = suite("Request")(
+    suite("toString") {
+      test("should produce string representation of a request") {
+        val r = Request()
```

Comment:
One small change.
Use ZIO.test.gen to generate values for `url`, `method`, `headers`, and `remoteAddress` and assert exact stringified request.
Refer [this](https://github.com/dream11/zio-http/blob/main/zio-http%2Fsrc%2Ftest%2Fscala%2Fzhttp%2Finternal%2FHttpGen.scala)